### ISSUE TITLE
how to just test one custom jpg image for gaze redirection ? 

### ISSUE THREAD

**Issue by A7med01** [AlessandroRuzzi/GazeNeRF#6]

**Comment by A7med01** [comment]
@X-Shi 
@AlessandroRuzzi 

any help with the steps ?

**Comment by X-Shi** [comment]
Hi, given one custom image is not enough to test gaze redirection. The gaze direction label for that image is also required. Suppose you have the image and the corresponding gaze direction label, you should preprocess the single image and then fine-tune the pre-trained model on it.
